shared/resources/styles/variables/openstax-bootstrap.less

```diff
@@ -191,13 +190,13 @@
 // @input-border:                   #ccc;
 
 // // TODO: Rename `@input-border-radius` to `@input-border-radius-base` in v4
-// //** Default `.form-control` border radius
-// // This has no effect on `<select>`s in some browsers, due to the limited stylability of `<select>`s in CSS.
-// @input-border-radius:            @border-radius-base;
-// //** Large `.form-control` border radius
-// @input-border-radius-large:      @border-radius-large;
-// //** Small `.form-control` border radius
-// @input-border-radius-small:      @border-radius-small;
+//** Default `.form-control` border radius
+// This has no effect on `<select>`s in some browsers, due to the limited stylability of `<select>`s in CSS.
+@input-border-radius:            0;
+//** Large `.form-control` border radius
+@input-border-radius-large:      0;
+//** Small `.form-control` border radius
+@input-border-radius-small:      0;
 
```

Comment:
I'm guessing UX is ok with removing the border-radius on all buttons?   If so we should we go through styles and find all the spots where we've already set it to zero and update them?